dashboard/app/models/code_review_comment.rb

```diff
@@ -8,7 +8,7 @@
 #  script_id        :integer
 #  level_id         :integer
 #  commenter_id     :integer          not null
-#  comment          :text(16777215)
+#  comment          :text(65535)
```

Comment:
Do you mind doing these as a separate PR? I don't see diffs locally when I migrate on staging but I do see diffs to these files on your branch. I don't know why these are being annotated differently in different environments, but it's probably good to work that out

Reply:
I think it's related to UTF8 encoding.  @jamescodeorg had some expertise in this area...

Reply:
Hmmm, I see this locally too and never dug into this.  I found this issue that's probably relevant: https://github.com/ctran/annotate_models/issues/278

Reply:
Looks like the annotate gem is [getting this value](https://github.com/ctran/annotate_models/blob/develop/lib/annotate/annotate_models.rb#L881) from rails.

```
Locally:
[development] dashboard > CodeReviewComment.columns[6]
=> #<ActiveRecord::ConnectionAdapters::MySQL::Column:0x00007f94f06c2aa0 @name="comment", @table_name="code_review_comments", @sql_type_metadata=#<ActiveRecord::ConnectionAdapters::SqlTypeMetadata:0x00007f94f06c2c80 @sql_type="text", @type=:text, @limit=65535, @precision=nil, @scale=nil>, @null=true, @default=nil, @default_function=nil, @collation="utf8mb4_bin", @comment=nil>

Staging:
[staging] dashboard > CodeReviewComment.columns[6]
=> #<ActiveRecord::ConnectionAdapters::MySQL::Column:0x00005641279b5320 @name="comment", @table_name="code_review_comments", @sql_type_metadata=#<ActiveRecord::ConnectionAdapters::SqlTypeMetadata:0x00005641279b4fd8 @sql_type="mediumtext", @type=:text, @limit=16777215, @precision=nil, @scale=nil>, @null=true, @default=nil, @default_function=nil, @collation="utf8mb4_bin", @comment=nil>
```

Interestingly, the comment column has a different type locally vs. on staging.  A quick google search didn't turn up why this might be the case. 

(Ok, I'm going to stop procrastinating now.  :-)